src/Tasks/ResolveProjectBase.cs

```diff
@@ -293,6 +293,7 @@ protected void AddSyntheticProjectReferences(string currentProjectAbsolutePath)
                     item.SetMetadata("ReferenceOutputAssembly", "false");
                     item.SetMetadata("LinkLibraryDependencies", "false");
                     item.SetMetadata("CopyLocal", "false");
+                    item.SetMetadata("SkipGetTargetFrameworkProperties", "false");
```

Comment:
Why not set this to `true`?

Reply:
Apparently skipping caffeine this morning wasn't my best idea.